tech_supp.qmd

```diff
@@ -294,18 +294,7 @@ C_{\text{slope}}(x) = \begin{cases}
 \end{cases}
 $$ {#eq-slopequant}
 
-| Slope (%) | Slope description     |
-|-----------|-----------------------|
-| \<3       | easy                  |
-| 3-5       | gentle slope          |
-| 5-10      | medium slope          |
-| 10-20     | steep slope           |
-| 20-30     | very steep slope      |
-| \>30      | extremely steep slope |
-
-: Six classification slopes used, adapted from @Dick2022
-
-==**TODO: Why are these intervals different from the ones above?**==
+These classification intervals differ from those used in @Dick2022 but are consistent with the [earliest known `R` implementation of the RP model](https://github.com/BioDT/uc-ces/blob/main/recreation_model/MODEL_Recreation_potential.R#L75).
```

Comment:
We should report on what is actually computed, not what *might* have been intended.

For what it's worth my guess is that these values have been modified by some software after being initially set by the user, perhaps during a reprojection operation. That aligns with a plausible explanation for the weird parameter values in the logistic function - see https://github.com/BioDT/uc-ces-recreation/issues/19#issuecomment-2999151758